cartridges/int_adyen_overlay/cartridge/scripts/util/AdyenHelper.ds

```diff
@@ -407,23 +407,63 @@ var __AdyenHelper: Object = {
         var shippingAddress = order.defaultShipment.shippingAddress;
         jsonObject["countryCode"] = shippingAddress.countryCode.value.toUpperCase();
 
+        var shippingStreet = "";
+        var shippingHouseNumberOrName = "";
+
+        if (!empty(shippingAddress.address1)) {
+            shippingStreet = shippingAddress.address1;
+
+            if (!empty(shippingAddress.address2)) {
+
+                if (paymentMethod.indexOf("afterpay_default") > -1) {
+                    shippingHouseNumberOrName = shippingAddress.address2
+                } else {
+                    shippingStreet += ' ' + shippingAddress.address2;
+                }
+            }
+
+        } else {
+            shippingStreet = 'N/A';
```

Comment:
It can happen that the address1 is empty but address2 is not so I would still append the address2 even if the address1 is empty

Reply:
Address 1 is a required field within the SFCC UI so it is not possible that this is empty...but I'll make the change anyway ;)

Reply:
in this case if SFCC handles it by default then I will mark it as resolved :)